tests/ceph_volume/test_ceph_volume_prepare_lvm.py

```diff
@@ -0,0 +1,74 @@
+import json
+import time
+
+from cli.cephadm.cephadm import CephAdm
+from cli.utilities.operations import wait_for_osd_daemon_state
+from cli.utilities.utils import create_yaml_config
+from utility.lvm_utils import lvm_create, pvcreate, vgcreate
+
+
+def _identify_disks(node):
```

Comment:
Same steps used here [1], better to move to [2]
[1] https://github.com/red-hat-storage/cephci/pull/3353/files#diff-8b844a814d2a59b8b0f3ce58511dd740498d9f407634069189809e3cfac77003R36
[2] https://github.com/red-hat-storage/cephci/blob/master/cli/utilities/utils.py

Reply:
Yeah, that will be good. I thought of doing it.

Reply:
If I import Cephadm in untils.py tox is complaining because cli/utilities/utils is imported in cephadm.py